### ISSUE TITLE
support for user-specific time zones

### ISSUE THREAD

**Issue by jaguarfi** [apluslms/a-plus#502]
> **Migrated from Trello**

---

**Conversation**

>**Jaakko Kantojärvi**
Is this really relevant?

>**Juha Sorva**
Not yet; I would postpone it for the future.

**Comment by jkotimak** [comment]
@jsorva is this any more relevant now?

**Comment by jsorva** [comment]
Not really. It was relevant for while when we had more international students taking the MOOC. But we haven’t really advertised the thing internationally, and there are few of those now. As far as I’m concerned, this issue can be closed. If it becomes relevant again, we can reopen it.
